### PR TITLE
[15.0][MIG] mrp_production_quant_manual_assign

### PR DESCRIPTION
**Pull request by ThiagoMForgeFlow**
Standard migration to 15.0

@ForgeFlow

### CODE REVIEW

**Review by DavidJForgeFlow — APPROVED** [review]
Functional review :+1: LGTM

File: mrp_production_quant_manual_assign/wizards/assign_manual_quants.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        if self._is_production_single_lot(move):
+            precision_digits = self.env["decimal.precision"].precision_get(
+                "Product Unit of Measure"
+            )
+            lots_to_consume = self.quants_lines.filtered(
+                lambda l: l.to_consume_now
+            ).mapped("lot_id")
+            for ml in move.move_line_ids:
+                if ml.lot_id in lots_to_consume:
+                    ml.qty_done = ml.product_qty
```
Comment (alexis-via) [inline mrp_production_quant_manual_assign/wizards/assign_manual_quants.py]: This is wrong ! qty_done is in product_uom_id whereas product_qty is in the product's unit of measure.
You should use "ml.product_uom_qty" which is in product_uom_id

File: mrp_production_quant_manual_assign/wizards/assign_manual_quants.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                elif float_is_zero(ml.product_qty, precision_digits=precision_digits):
```
Comment (alexis-via) [inline mrp_production_quant_manual_assign/wizards/assign_manual_quants.py]: It would be better to use ml.product_uom_qty

**Review by alexis-via — CHANGES_REQUESTED** [review]

File: mrp_production_quant_manual_assign/wizards/assign_manual_quants.py (hunk 1 above)
Comment (alexis-via) [reply mrp_production_quant_manual_assign/wizards/assign_manual_quants.py]: By the way, the bug has been there for quite some time. I just reported it on stock_quant_manual_assign https://github.com/OCA/stock-logistics-warehouse/issues/1527

Comment (ThiagoMForgeFlow) [reply mrp_production_quant_manual_assign/wizards/assign_manual_quants.py]: Done

File: mrp_production_quant_manual_assign/wizards/assign_manual_quants.py (hunk 2 above)
Comment (ThiagoMForgeFlow) [reply mrp_production_quant_manual_assign/wizards/assign_manual_quants.py]: Done

**Review by alexis-via — APPROVED** [review]

**Pull request merged**